src/components/Pricing/Addons/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { IconExternal, IconInfo, IconGroups, IconProfile, IconDecisionTree, IconBuilding } from '@posthog/icons'
+import { useValues } from 'kea'
+import React, { useEffect, useState } from 'react'
+import Link from 'components/Link'
+import Tooltip from 'components/Tooltip'
+import cntl from 'cntl'
+
+export const section = cntl`
+    max-w-6xl
+    xl:max-w-7xl
+    mx-auto
+    px-4
+`
+
+const addons = [
```

Comment:
Just noting that we're gonna want to find a way to get these from the API.

Reply:
Yep, was just focused on building out the HTML here. Source the data however you want!

Reply:
Can we do this now? Or do we actually want to do it later for some reason?

Reply:
Yes I will work on this tomorrow.